core/src/main.cpp

```diff
@@ -1064,6 +1108,10 @@ auto main(int _argc, char* _argv[]) -> int
 		logging::trace("Waiting for I/O thread pool to shut down.");
 		io_threads.join();
 
+		logging::debug("Cleaning up user manager...");
```

Comment:
trace here - debug usually shows a variable/value of some kind

Reply:
Done.